.circleci/config.yml

```diff
@@ -39,79 +39,35 @@ commands:
       - aws-s3/copy:
           from: << parameters.filename >>
           to: << parameters.bucket >>
-          arguments: '--acl public-read --cache-control no-cache'
-
-  install-golangci-lint:
```

Comment:
Why is this no longer necessary?

Reply:
Because `golangci` is already installed? Or did I miss the point of that step?